client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/IssueService.java

```diff
@@ -70,6 +70,22 @@ public interface IssueService {
   @JsonRequest
   CompletableFuture<Void> addComment(AddIssueCommentParams params);
 
+  /**
+   * Checks if the anticipated transitions are supported
+   * <ul>
+   *   <li>The provided connection should link to a SonarQube 10.2+ instance</li>
+   * <p>
+   * This method will fail if:
+   * <ul>
+   *   <li>the connectionId provided as a parameter is unknown</li>
+   *   <li>there is a communication problem with the server: network outage, server is down, unauthorized</li>
+   * </ul>
+   * In those cases, a failed future will be returned.
+   * </p>
+   */
+  @JsonRequest
+  CompletableFuture<CheckAnticipatedStatusChangeSupportedResponse> checkAnticipatedStatusChangeSupported(CheckAnticipatedStatusChangeSupportedParams params);
```

Comment:
The name "anticipated status or transition" is the name of the feature and endpoint on the SQ side. We decided to call the feature "resolving local-only issues" because it makes more sense from the SL point of view. But maybe not all our code base uses this name